### DISCUSSION TITLE
fonts for vscode

### DISCUSSION THREAD

**Discussion by FernandaMaria28** [Schweinepriester/github-profile-achievements#493] [Q&A] [answered]
What text fonts recommend?

**Comment by bawejagb** [comment, marked as answer]
I recommend you to use the **JellyFish Theme** Extention, it has very great text fonts. ❤️‍🔥
Must Try, I've been using it for the past few months, and I love it.
Link: [https://marketplace.visualstudio.com/items?itemName=PawelBorkar.jellyfish](url)

**Comment by abidurrahman11** [comment]
You can use consolas font, as it comes as default font with most of the code editors. 😀

**Comment by khatrisagar** [comment]
i am using default font which is better.
